### PR TITLE
Small Uncertainty changes

### PR DESCRIPTION
**Pull request by FeGeyer**
Small fixes for uncertainty training:

- Smaller value for GeneralELU
- splitted L1 loss functions for uncertainty training with weight on phase
- fix `get_prediction` for combined architectures

### CODE REVIEW

File: radionets/dl_framework/loss_functions.py [hunk 1]
```diff
@@ -204,14 +230,10 @@ def comb_likelihood(x, y):
...
     loss_amp = (
-        0.5 * torch.log(amp_unc.pow(2))
-        + ((y_amp - amp_pred).pow(2) / amp_unc.pow(2))
-        + (y_amp - amp_pred)
+        2 * torch.log(amp_unc) + ((y_amp - amp_pred).pow(2) / amp_unc.pow(2))
     ).mean()
     loss_phase = (
-        0.5 * torch.log(phase_unc.pow(2))
-        + ((y_phase - phase_pred).pow(2) / phase_unc.pow(2))
-        + (y_phase - phase_pred)
```
Comment (Kevin2) [inline radionets/dl_framework/loss_functions.py:214]: when did we add the addiotional +(y -pred) term?

Comment (FeGeyer) [reply radionets/dl_framework/loss_functions.py:214]: That was you three months ago [here](https://github.com/radionets-project/radionets/commit/4adbc445f3b0077d0e154ab09b1fd6f50eb463dc#). I do not know why though.

Comment (Kevin2) [reply radionets/dl_framework/loss_functions.py:214]: Ok, we drop it for now

**Review by Kevin2 — APPROVED** [review]
